Backend/Data/ProductList.js

```diff
@@ -0,0 +1,2191 @@
+export const ProductList = [
+    {
+        id: 1,
```

Comment:
Hey Shivam
Thank you for your review,
I kept product details locally too, because i was afraid if something happens to my data in database then i would loose all data so,
Appreciate your suggestion
Thank you
Akshay